app/src/main/java/org/fossasia/openevent/general/event/EventDetailsFragment.kt

```diff
@@ -0,0 +1,87 @@
+package org.fossasia.openevent.general.event
+
+import android.arch.lifecycle.Observer
+import android.os.Bundle
+import android.support.v4.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import android.widget.Toast
+import com.squareup.picasso.Picasso
+import kotlinx.android.synthetic.main.content_event.view.*
+import org.fossasia.openevent.general.R
+import org.koin.android.architecture.ext.viewModel
+import timber.log.Timber
+
+class EventDetailsFragment : Fragment() {
+    private val eventViewModel by viewModel<EventDetailsViewModel>()
+    private lateinit var rootView: View
+    private var eventId : Long = 0
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        val bundle = this.arguments
+        if (bundle != null) {
+            eventId = bundle.getLong("EVENT_ID", 0)
```

Comment:
Default should be - 1 and if it is, then the activity should return

Reply:
okay